packages/main-library/package.json

```diff
@@ -16,7 +16,7 @@
     "test": "jest --coverage"
   },
   "dependencies": {
-    "xlsx": "^0.18.5"
+    "xlsx-js-style": "^1.2.0"
```

Comment:
Regarding this I am not sure if it's okay that we change `xlsx` to `xlsx-js-style`, I think it's better to have both and allow devs to choose if they want to use one or other

So we can use `xlsx(data, settings, useStyleLibrary = false)`

Reply:
In the xlsx-js-style library, the xlsx dependency also has version 0.18.5, and styling is optional, you can simply not use the styling syntax